### DISCUSSION TITLE
[Awsome] we can learn from these awsome projects 👍

### DISCUSSION THREAD

**Discussion by tyn1998** [hypertrons/hypertrons-crx#652] [Ideas]
### Description

- https://github.com/refined-github/refined-github
- https://github.com/BloopAI/bloop
- https://chatdoc.com/

**Comment by andyhuang18** [comment]
I have installed ```refined-github``` plugin. I can easily see which users responded to the emoji, which is by far my favourite thing about it.

**Comment by will-ww** [comment]
> I have installed `refined-github` plugin. I can easily see which users responded to the emoji, which is by far my favourite thing about it.

+1  :-)
